hawc/apps/assessment/widgets.py

```diff
@@ -0,0 +1,40 @@
+from django import forms
+from django.utils import timezone
+
+
+class DateCheckboxInput(forms.widgets.Input):
```

Comment:
why not override the checkbox input?

Reply:
I thought I was going to have to replace more code than I actually ended up doing. I'll go back and change that, I'm having some formatting issues right now so that may actually fix it